wakuv2/waku.go

```diff
@@ -1092,9 +1090,12 @@ func (w *Waku) Start() error {
 		}
 	}()
 
+	w.wg.Add(1)
```

Comment:
we've got `services/wallet/async/async.go` `Group` for managing this repeating pattern of adding 1 to the WaitGroup and running a thingy in a goroutine. Kind of a small pattern, but it should prevent the kind of bug where you forget to add 1.

Reply:
@dlipicar Not sure about this. `WaitGroup` itself is a pretty simple class and straight-forward to use. And wrapping it into such things unnecessary complicates it IMO.